setup.py

```diff
@@ -47,6 +47,8 @@ def package_files(directory):
         'entrypoints',
         'sqlparse',
         'sqlalchemy',
+        'docker>=3.6.0',
```

Comment:
Can we remove these from the install_requires? Not everyone who uses mlflow will use k8s projects so we shouldn't force these deps on them.

Reply:
I see. Okay. Removed.